upa/src/tool/log.ts

```diff
@@ -1,21 +1,54 @@
 import { Logger, createLogger, format, transports } from "winston";
 import * as process from "node:process";
+import LokiTransport from "winston-loki";
+import Transport from "winston-transport";
 
 let logger: Logger | undefined;
 
 export function getLogger() {
   if (!logger) {
-    const level = process.env["UPA_LOG_LEVEL"] || "info";
+    let processBinName = process.argv[1].split("/").pop();
+    // Safety measure in case above returns undefined
+    processBinName = processBinName ? processBinName : "upa";
+    // Replace hyphens with underscores
+    const job = processBinName.replace(/-/g, "_");
+    const level = process.env[`${job}_LOG_LEVEL`] || "info";
```

Comment:
Happy to keep this as just `LOG_LEVEL` across all services to make things simpler.  Up to you, though.

Reply:
Done.